docs/api.md

```diff
@@ -2,8 +2,22 @@
 
 auro-badge provides users a way to ...
 
+## Attributes
+
+| Attribute  | Type     | Description                                  |
+|------------|----------|----------------------------------------------|
+| `cssClass` | `String` | Applies designated CSS class to DOM element. |
+
 ## Properties
 
-| Property   | Attribute  | Type     | Description                                  |
-|------------|------------|----------|----------------------------------------------|
-| `cssClass` | `cssClass` | `String` | Applies designated CSS class to DOM element. |
+| Property | Attribute | Type         | Default      |
+|----------|-----------|--------------|--------------|
+| `action` | `action`  | `boolean`    |              |
+| `dom`    |           |              |              |
```

Comment:
This and the following appear to be private properties and methods. See https://github.com/AlaskaAirlines/auro-icon/pull/15/files#diff-82293b9915453faa44b6cc3cffe36b2e4e83136a64b9197bf067efd1b300ba72 for some examples of how to hide this from the API doc.